src/features/game/types/images.ts

```diff
@@ -227,1523 +21,66 @@ import kaleMushroomPie from "assets/food/mushroom_kale_pie.png";
-    description: ACHIEVEMENTS()["Fruit Platter"].description,
-  },
-  "Ayam Cemani": {
-    image: ayamCemani,
-    description: "The rarest chicken alive!",
-  },
-  "Collectible Bear": {
-    image: collectibleBear,
-    description: "A prized bear, still in mint condition!",
-  },
-  "Cyborg Bear": {
-    image: cyborgBear,
-    description: "Hasta la vista, bear",
-  },
-  "Maneki Neko": {
-    image: manekiNeko,
-    description: "The beckoning cat. Pull it's arm and good luck will come",
-  },
-  "Red Envelope": {
-    image: redEnvelope,
-    description: "Wow, you are lucky!",
-  },
-  "Abandoned Bear": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Crop Charm": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Fossil 1": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Fossil 2": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Fossil 3": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Goblin Bear": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Golden Bear Head": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Human Bear": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Lifeguard Bear": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Pirate Bear": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Pirate Bounty": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Pirate Cake": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Pirate Hat": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Skeleton King Staff": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Snorkel Bear": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Sunflower Coin": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Treasure Map": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Turtle Bear": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  "Wood Charm": {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  Foliant: {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  Galleon: {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  Pearl: {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  Pipi: {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
-  Seaweed: {
-    image: SUNNYSIDE.icons.expression_confused,
-    description: "?",
-  },
+export const IMAGES: Record<number, ItemDetailsWithImages> = {
```

Comment:
What is the goal of indexing images by ID?

From a developer experience POV, we want to avoid using IDs where possible and use the string representation.

E.g.

`Pumpkin Soup`: pumpkinSoup.

This makes it much easier to read and follow throughout the code base